staging/src/k8s.io/component-helpers/storage/volume/helpers.go

```diff
@@ -16,7 +16,14 @@ limitations under the License.
 
 package volume
 
-import v1 "k8s.io/api/core/v1"
+import (
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/component-helpers/scheduling/corev1"
```

Comment:
cross-package use of helpers is unexpected... should this function just live in `k8s.io/component-helpers/scheduling/corev1`?

Reply:
fine with me to have it in scheduling

Reply:
If we do that, `pkg/volume/util/operationexecutor` will depends on `k8s.io/component-helpers/scheduling/corev1`. Is it ok?

Reply:
I think the way the dependencies are working now is good.